docs/sources/search/_index.md

```diff
@@ -83,3 +83,22 @@ To filter dashboard search result by a tag, complete one of the following steps:
   All tags will be shown, and when you select a tag, the dashboard search will be instantly filtered.
 
 > **Note:** When using only a keyboard, press the `tab` key and navigate to the **Filter by tag** drop-down menu, press the down arrow key `▼` to activate the menu and locate a tag, and press `Enter` to select the tag.
+
+# Command Palette
+
+Open the Command Palette by pressing `cmd+K` in macOS or `ctrl+k` in Linux/Windows. You can also click on the input placed in the navbar.
+
+![Command Palette screenshot](/static/img/docs/command-palette/CommandPalette_doc_1.png)
+
+By typing in this field, you can:
+
+1. Search and open dashboards and folders.
+2. Create dashboards and alert rules.
+3. Navigate to various areas or pages.
+4. Change the theme: dark or light.
+
+![Select change theme](/static/img/docs/command-palette/CommandPalette_doc_4.png)
```

Comment:
Thoughts on images:
1) We are trying to limit them as much as possible because they become tech debt. If you could only have 1 screenshot, which one would it be?
2) images should now be stored in the GCS bucket located here: https://admin.grafana.com/upload/media/docs/grafana/

Reply:
Thanks @chri2547! I will keep the first one. It makes more sense to show a generic image if we can only add one.